jujugui/static/gui/src/app/components/entity-details/entity-details.js

```diff
@@ -18,14 +18,111 @@ with this program.  If not, see <http://www.gnu.org/licenses/>.
 
 'use strict';
 
-YUI.add('entity-details', function() {
+YUI.add('entity-details', function(Y) {
+
+  var utils = Y.namespace('juju.views.utils');
```

Comment:
No YUI.

I commented below in the utils file about creating a component utils file. Because it would be a utils file it would probably be nice to pass it to every top level component.